src/index.ts

```diff
@@ -71,11 +72,15 @@ export class Probot {
     // Log all webhook errors
     this.webhook.on('error', this.errorHandler)
 
-    if (process.env.REDIS_URL) {
+    if (options.redisConfig || process.env.REDIS_URL) {
       const Bottleneck = require('bottleneck')
```

Comment:
I know this was added before, but would you mind moving this up to the top of the module and doing `import Bottleneck from 'bottleneck'` so we get type completion? Written like this all the bottleneck objects are just 'any'. Everything else looks good to me!